docs/source/pages/howToWorkWithPlatform/adminApplication/users/users.rst

```diff
@@ -2,173 +2,191 @@
 
-For select permission to mailbox need go to (1) mailbox tab. On this tab presented mailboxes which related to this account.
+Grant mailbox permissions
+=========================
 
-.. image:: pic_users/users_5.png
-   :width: 600
+You can grant mailbox roles and permissions in a similar manner as account roles and permissions on the mailbox tab. Only difference is that you need to click the required mailbox first, and then selects roles and/or permissions.
+
+.. image:: pic_users/mailboxRoles.png
+   :width: 400
    :align: center
 
-To add permissions to mailbox you can in the same way as to account. The main difference is we have to choose a mailbox to which permissions will be added. If you have defined all permission which you want to provide to user click the "Invite" button in the header. If invite has created successfully at the top right corner of the page notification message will be shown, and invitation letter will be sent to the email which has been provided. The process of the invitation will be evolved in two directions, depending on whether we are inviting new or existing user.
+.. image:: pic_users/mailboxPermissions.png
+   :width: 400
+   :align: center
 
-.. note:: `Behaviour of invite internal user <usersInviteInternal.html>`_
+.. note:: :ref:`Mailbox permissions list and description <mailboxPermissions>`
 
-.. note:: `Behaviour of invite external user <usersInviteExternal.html>`_
+Registered user invitation
+==========================
 
-.. _createUsers:
+After an invite is sent, registered user will get an email with a link to the invites tab of user profile page with all invites listed there and options to accept or decline them.
 
-Create one user
-===============
+.. image:: pic_users/invitesTab.png
+   :width: 400
+   :align: center
 
-As administrator of account you can create users with verified and activated domain. It can be used if you working in company and you have to create corporate users. How to add, verify and activate domain you can find by link :ref:`domains configuration <domains>`. To create corporate user you have to do the following:
+After user accepted or declined an invitation you will receive a notification email with details.
 
-1. Navigate to https://platform_address_here/admin/users with administratore access
-2. Click the "Create user" button
+Not registered user invitation
+==============================
 
-.. image:: pic_users/usersPage.png
-   :width: 600
-   :align: center
+After an invite is sent, not registered user will receive an email with a link to the registration page. By following the link user will be redirected to the registration form with prefilled email address and name (if it was filled in the invitation form). After completing this form, profile, account and personal mailbox (alongside with access to the mailboxes from the invitation) will be created for the user and permissions specified in the invitation will be granted.
```

Comment:
prefilled email address and name (if it was filled in the invitation form).

I suppose we've discussed case where we're not fill in email/name is impossible according to BE validations